src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -377,14 +389,15 @@ export class BrowsingContextImpl {
         if (!['default', 'isolated'].includes(params.context.auxData.type)) {
           return;
         }
-        const realm = Realm.create(
+        const realm = new Realm(
+          this.#realmStorage,
           params.context.uniqueId,
           this.contextId,
           this.navigableId ?? 'UNKNOWN',
           params.context.id,
           this.#getOrigin(params),
           // TODO: differentiate types.
-          RealmType.window,
+          'window',
```

Comment:
should this remain a named constant?

Reply:
RealmType.window causes a compilation error